assets/src/scripts/monitoring-location/components/hydrograph/method-picker.js

```diff
@@ -25,32 +55,8 @@ export const drawMethodPicker = function(elem, store, initialTimeSeriesId) {
     pickerContainer.append('select')
         .attr('class', 'usa-select')
         .attr('id', 'method-picker')
+        .call(link(store, updateAvailableMethods, getPrimaryMethods, store))
```

Comment:
I pulled this code into a named function. I think we use anonymous functions too often in our rendering code (I have been as guilty of this as anyone else). The anonymous function on line 46 is fine because it's a one line function and what it does is pretty clear. But this one is more complicated and I think worthy of being a named function.